docs/reference/authorization.md

```diff
@@ -47,7 +47,7 @@ See [this example for authenticating with the AWS CLI](../using/aws_cli.md).
 
 Access to resources is managed very much like [AWS IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/intro-structure.html){:target="_blank"}.
 
-There are 4 basic components to the system:
+There are 4 basic components of the system:
```

Comment:
Please revert this change, "to" is correct here and is common.  In this sentence "of" would imply that there can be other components.

Reply:
done